### PR TITLE
printer_mods/Mini_After-LGX-Lite

### PR DESCRIPTION
**Pull request by lejo1512**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ like so:
 | bobbleheed | [Magnetic Panels](./bobbleheed/Magnetic_Panels) | Magnetic clips to allow for easy removal and installation of panels | :x: | :grey_question: | :heavy_check_mark: | :x: | :grey_question: |
 | | [Filament Runout Sensor](./bobbleheed/Filament_Runout) | Filament runout sensor that replaces the stock bowden retainer | :x: | :heavy_check_mark: |:heavy_check_mark: | :x: | :heavy_check_mark: |
 |Boingomw|[Wago mount](./Boingomw/Wago_mount)| Smaller 5 port Wago mount that fits parallel to the support rails |:x:|:grey_question:|:heavy_check_mark:|:x:|:grey_question:|
+|Bondtech|[Mini_After-LGX-Lite](./Bondtech/Mini_After-LGX-Lite) |LGX Lite Toolhead Setup For Voron V0.1. | :heavy_check_mark: | :x: | :x: |:x:|:x:|
```
Comment (cbc02009) [inline printer_mods/README.md:75]: Are you an official representative of bondtech? otherwise put this mod under your own username.

**Review by cbc02009 — CHANGES_REQUESTED** [review]
there are bunch of extra files added to the base printer_mods folder that need to be removed.

File: printer_mods/README.md (hunk 1 above)
Comment (lejo1512) [reply printer_mods/README.md:75]: Yes I'm an official representative of Bondtech.

**Review by cbc02009 — CHANGES_REQUESTED** [review]
printer_mods/Bondtech/Mini_After-LGX-Lite/STL/Z_Bed adjustable Y offset/

Sorry I missed it in the last review, but it looks like there's spaces in one of the folder names. once that's fixed this should be good to merge in.

**Review by cbc02009 — APPROVED** [review]

**Comment by cbc02009** [comment]
Looks like the CI found some STL issues that need fixing. You can see the list here: https://github.com/VoronDesign/VoronUsers/pull/623/checks

**Comment by cbc02009** [comment]
```  Validating STL printer_mods/Bondtech/Mini-After-LGX-Lite/STL/Mini-After-LGX-lite/hotend_mount_copperhead_screw_mount_3007_fan.stl
  !! Detected "Facets added" value of 2
  Validating STL printer_mods/Bondtech/Mini-After-LGX-Lite/STL/Mini-After-LGX-lite/hotend_mount_copperhead_screw_mount_3010_fan.stl
  !! Detected "Facets added" value of 2
  Validating STL printer_mods/Bondtech/Mini-After-LGX-Lite/STL/Mini-After-LGX-lite/hotend_mount_dragon_3007_fan.stl
  !! Detected "Edges fixed" value of 4
  Validating STL printer_mods/Bondtech/Mini-After-LGX-Lite/STL/Mini-After-LGX-lite/hotend_mount_dragonfly_3007_fan.stl
  !! Detected "Edges fixed" value of 4
  Validating STL printer_mods/Bondtech/Mini-After-LGX-Lite/STL/Mini-After-LGX-lite/hotend_mount_dragonfly_3010_fan.stl
  !! Detected "Facets added" value of 2
  Validating STL printer_mods/Bondtech/Mini-After-LGX-Lite/STL/Mini-After-LGX-lite/hotend_mount_mosquito_3007_fan.stl
  !! Detected "Edges fixed" value of 4
  Validating STL printer_mods/Bondtech/Mini-After-LGX-Lite/STL/Mini-After-LGX-lite/hotend_mount_mosquito_3010_fan.stl
  !! Detected "Facets added" value of 2``` Doesn't look like the STL issues were all fixed.

**Comment by cbc02009** [comment]
Also x-carriage.stl is not printable on an FDM printer without supports. either modify it so that it will print, or add removable supports to the STL.

**Comment by lejo1512** [comment]
There are 3 versions of the x-carriage one of them is tagged with “supported”. Should I remove the other FDM version then? The third version is for SLS so no support. 
 
Thank you for your time!

**Comment by cbc02009** [comment]
I would remove the unsupported one, or clearly label it as unsupported.

**Review by cbc02009 — APPROVED** [review]

**Comment by cbc02009** [comment]
https://github.com/VoronDesign/VoronUsers/pull/623/checks#step:7:351 looks like there's a bad link in the readme file.

**Pull request merged**